classes/Divisions.php

```diff
@@ -179,6 +179,32 @@ public function getAllMemberDivisionsByPolicy() {
     }
 
 
+    /**
+     * Get the last n votes for a member
+     *
+     * Returns an array of divisions
+     */
+    public function getRecentMemberDivisions($number = 20) {
+        $args = array(':person_id' => $this->member->person_id, ':number' => $number);
+        $q = $this->db->query(
+            "SELECT division_id, division_title, yes_text, no_text, division_date, division_number, vote, gid, direction
```

Comment:
Not that it really matters, but it looks like you don't need the `division_number` column here.